controller/token.go

```diff
@@ -231,6 +231,9 @@ func (c *TokenController) Exchange(ctx *app.ExchangeTokenContext) error {
 func (c *TokenController) exchangeWithGrantTypeRefreshToken(ctx *app.ExchangeTokenContext) (*app.OauthToken, error) {
 	// retrieve the access token from the request header, but ignore if it was not found
 	accessToken := goajwt.ContextJWT(ctx)
+	if accessToken == nil {
+		return nil, errors.NewBadParameterError("access_token", nil).Expected("not nil")
```

Comment:
Hm.. According to the comment above (line 232) Access Token is optional but it's actually not optional since it's used in line 251. I would delete this change from this PR and address it in a different PR to make sure we are not breaking any API.
Also the error message is not correct. It's not a parameter we expect here. It's `Authorization` header. Would need to use `errors.NewBadParameterErrorFromString()` instead.